src/types/gecko-profile.js

```diff
@@ -130,12 +130,22 @@ export type GeckoThread = {
   stringTable: string[],
 };
 
+export type GeckoExtensionMeta = {|
+  schema: {|
+    id: 0,
+    name: 1,
+    baseURL: 2,
+  |},
+  data: Array<[string, string, string]>,
+|};
+
 export type GeckoProfile = {|
   meta: {|
     interval: Milliseconds,
     startTime: Milliseconds,
     shutdownTime: Milliseconds | null,
     abi: string,
+    extensions?: GeckoExtensionMeta,
```

Comment:
Could you add this comment here to clarify why it's optional? I'm assuming that newer gecko profiles will always have this property.

```
// The extensions property landed in Firefox 60, and is only optional because older profile versions
// may not have it. No upgrader was written for this change.
```